spec/requests/resource_operations_spec.rb

```diff
@@ -70,7 +70,7 @@
   end
 
   describe 'POST /articles' do
-    subject(:last_response) { post("/articles", '{ "data": { "type": "articles" } }') }
+    subject(:last_response) { post("/articles", '{ "data": { "id": "1", "type": "articles" } }') }
```

Comment:
I don't quite follow why you must specify an `id` here? Is it that this PR changes the `Article` resource to have its ID generated client-side like as an UUID? Auto-increment-like behavior no longer works, I assume.